config/settings.yml

```diff
@@ -143,6 +143,7 @@ paperclip:
   path: ":rails_root/public/:attachment/:id_partition/:style/:safe_filename"
 
 # saml:
+#   create_user: false
```

Comment:
I feel like the "default" we show should be the actual default. Or we should add some documentation around it.

If we start wanting to add more documentation, then maybe we do a thing in the engine where we add another Settings source, like a `saml.yml`.

Reply:
A `saml.yml` file is an interesting idea. I don't know if we have enough saml-specific stuff to justify that yet, but good thing to keep in mind going forward.